WWW/smoorthi-dev/scatter_gather/minidata_mutation_calling_v3.wdl

```diff
@@ -306,7 +324,8 @@ task ApplyBaseRecalibrator {
     referenceGenome refGenome
   }
   
-  String base_file_name = basename(input_bam, ".duplicates_marked.bam")
+  String base_file_name = basename(input_bam, ".bam")
+  String base_file_name_output_name = sub(base_file_name,"_.*", "")
```

Comment:
I see that you need to create base_file_name_output_name for an input for gatherBams, but this seems a little bit risky. It seems to be relying on a particular file naming scheme to work properly. Sometimes that's strictly necessary, but if you're relying on a naming scheme, you should tell users that so they can make sure their fastqs are name appropriately.

Reply:
So with that. This isnt the cleanest way! I agree but I want to keep the basefile name as only the sample name. To prevent the final file reading like HC4006.recal_bam.duplicates_marked......